packages/tasit-identity-contract/src/GnosisSafeUtils.js

```diff
@@ -137,13 +138,8 @@ export default class GnosisSafeUtils {
     }
   };
 
-  encodeFunctionCall = (contract, functionName, args) => {
-    const { getAddress, getABI, _getProvider } = contract;
-    const ethersContract = new ethers.Contract(
-      getAddress(),
-      getABI(),
-      _getProvider()
-    );
-    return ethersContract.interface.functions[functionName].encode(args);
+  encodeFunctionCall = (abi, functionName, args) => {
+    const contractInterface = new Interface(abi);
```

Comment:
This is a nice addition

Reply:
@marcelomorgado We can resolve this if you like

Reply:
Resolving